core/chains/evm/config/toml/defaults/fallback.toml

```
@@ -60,6 +60,11 @@ TransactionPercentile = 60
 [GasEstimator.FeeHistory]
 CacheTimeout = '10s'
 
+[GasEstimator.DAOracle]
+OracleType = ''
+OracleAddress = '0x0000000000000000000000000000000000000000'
+CustomGasPriceCalldata = ''
```

Are these empty strings meaningful values?
```suggestion
OracleAddress = '0x0000000000000000000000000000000000000000'
```
What about the zero value address?
```suggestion
```
Notice this file does not contain any other field with zero values.

---

yeah I noticed that as well, I just removed this addition and went with a different approach and updated `core/config/docs/docs_test.go`